purbee_backend/backend_source/app.py

```diff
@@ -27,6 +27,24 @@
 app = Flask(__name__)
 
 
+@app.route('/api/community_feed/', methods=['GET'])
+def community_feed():
+    req = request.get_json()
+    data = {"response_message": None}
+    status_code = None
+    community_id = req["community_id"]
+    community = Community.get_community_from_id(community_id)
+    if not community :
+        data['response_message'] = "there is no such community."
+        status_code = SC_FORBIDDEN
+        return data, status_code
+
+    data['response_message'] = "community post list successfully returned"
+    data['community_post_list'] = community.post_history_id_list.reverse()
+    status_code = SC_SUCCESS
+    return data, status_code
+
```

Comment:
I believe some cases are missing: 
* Is there any community with the given <coommunity_id>
* Is the community is public or private, if private does the user have an access to view this community's post history
* Is there any registered user with the given <user_id>

Reply:
I have implemented this cases. Please check again.